pkg/arvo/sys/hoon.hoon

```diff
@@ -6124,54 +6136,47 @@
     ==
   ++  royl
     ~+
-    ;~  pose
-      (stag %rh royl-rh)
-      (stag %rq royl-rq)
-      (stag %rd royl-rd)
-      (stag %rs royl-rs)
-    ==
-  ::
-  ++  royl-rh  (cook rylh ;~(pfix ;~(plug sig sig) (cook royl-cell royl-rn)))
-  ++  royl-rq  (cook rylq ;~(pfix ;~(plug sig sig sig) (cook royl-cell royl-rn)))
-  ++  royl-rd  (cook ryld ;~(pfix sig (cook royl-cell royl-rn)))
-  ++  royl-rs  (cook ryls (cook royl-cell royl-rn))
-  ::
-  ++  royl-rn
-    =/  moo
-      |=  a=tape
+    =+  ^=  moo
```

Comment:
These changes to `+royl` are a regression, please revert.

Reply:
ahh sorry about that one. will do.